### DISCUSSION TITLE
Any plans to support async/await?

### DISCUSSION THREAD

**Discussion by synackSA** [PyMySQL/PyMySQL#999] [Ideas]
As per the title. I was wondering if there are any plans in the near future to support async/await. This is becoming quite an important feature to support as many companies are moving in this direction to increase request times, especially in a microservices environment.

**Comment by cakemanny** [comment]
I'm not from PyMySQL, but I wouldn't have thought there would be a lot of merit in doing the work, given [`aiomysql`](https://github.com/aio-libs/aiomysql#aiomysql) exists and has done the work that needs to be done?
> Internally **aiomysql** is copy of PyMySQL, underlying io calls switched to async
